source/server/getting-started/configuration/sponge-conf.rst

```diff
@@ -13,464 +13,1165 @@ Below is a list of available config settings and their comments inside the globa
-    execution root or specified as an absolute path.
-    Note that the default: "${CANONICAL_GAME_DIR}/config"
-    is going to use the "plugins" directory in the root game directory.
-    If you wish for plugin configs to reside within a child of the configuration
-    directory, change the value to, for example, "${CANONICAL_CONFIG_DIR}/sponge/plugins".
-    Note: It is not recommended to set this to "${CANONICAL_CONFIG_DIR}/sponge", as there is
-    a possibility that plugin configurations can conflict the Sponge core configurations.
-disable-warnings
-    Disable warning messages to server admins
-file-io-thread-sleep
-    If 'true', sleeping between chunk saves will be enabled, beware of memory issues.
-plugins-dir
-    Additional directory to search for plugins, relative to the
-    execution root or specified as an absolute path.
-    Note that the default: "${CANONICAL_MODS_DIR}/plugins"
-    is going to search for a plugins folder in the mods directory.
-    If you wish for the plugins folder to reside in the root game
-    directory, change the value to "${CANONICAL_GAME_DIR}/plugins".
-
-logging
-=======
-
-block-break
-    Log when blocks are broken
-block-modify
-    Log when blocks are modified
-block-place
-    Log when blocks are placed
-block-populate
-    Log when blocks are populated in a chunk
-block-tracking
-    Log when blocks are placed by players and tracked
-chunk-gc-queue-unload
-    Log when chunks are queued to be unloaded by the chunk garbage collector.
-chunk-load
-    Log when chunks are loaded
-chunk-unload
-    Log when chunks are unloaded
-entity-collision-checks
-    Whether to log entity collision/count checks
-entity-death
-    Log when living entities are destroyed
-entity-despawn
-    Log when living entities are despawned
-entity-spawn
-    Log when living entities are spawned
-entity-speed-removal
-    Whether to log entity removals due to speed
-exploit-itemstack-name-overflow
-    Log when server receives exploited packet with itemstack name exceeding string limit.
-exploit-respawn-invisibility
-    Log when player attempts to respawn invisible to surrounding players.
-exploit-sign-command-updates
-    Log when server receives exploited packet to update a sign containing commands from player with no permission.
-log-stacktraces
-    Add stack traces to dev logging
-world-auto-save
-    Log when a world auto-saves its chunk data. Note: This may be spammy depending on the auto-save-interval configured for world.
-
-metrics-collection
+**broken-network-handler-mods**
+
+| A list of mod ids that have broken network handlers (they interact with the game from a Netty handler thread).
+| All network handlers from a forcibly scheduled to run on the main thread.
+| Note that this setting should be considered a last resort, and should only be used as a stopgap measure while waiting for a mod to properly fix the issue.
+
+* **Type:** ``List<String>``
+
+BungeeCordCategory
```

Comment:
Do we really need the word "Category" appended on this title? It's an artifact of the process

Reply:
The `Category` suffix is part of the actual class name.

Reply:
Indeed, but it's not used in the config file, so it might confuse people searching. It's not a major problem.

Reply:
I added clickable links, which should alleviate this issue.

Reply:
After further consideration, I removed the suffix now.